WooCommerce/Classes/Extensions/TimeZone+Woo.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+
+/// TimeZone: Constant Helpers
+///
+extension TimeZone {
+
+    /// Returns the TimeZone using the timezone configured in the current website
+    ///
+    static var websiteTimezone: TimeZone {
```

Comment:
nit: how about having the extension on the `Site` so that it's easier to test with different `gmtOffset`/`timezone` properties? and would be nice to have some unit tests on this 😄